api/v1beta1/disruption_spec.go

```diff
@@ -20,8 +20,11 @@ package v1beta1
 import (
 	"crypto/md5"
 	"encoding/json"
+	"errors"
```

Comment:
Everything looks great, except for some small changes to this file that I commented on in #320, though I can move those comments here if that's more helpful

Reply:
No need, I'll fix things asap, retest then merge